public/components/metrics/top_menu/metrics_export.tsx

```diff
@@ -50,32 +51,49 @@ const Savebutton = ({
 }) => {
   return (
     <EuiButton
+      size="s"
```

Comment:
I think based on mocks, we should add. `fill` prop here.

Reply:
fixed